src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,30 @@ const Cashier = (() => {
         });
     };
 
+    const showP2PVisibility = async () => {
+        const is_agent = !(await BinarySocket.send({ p2p_agent_info: 1 })).error;
+        if (is_agent) {
+            $('#dp2p_info').setVisibility(true);
```

Comment:
generally we use 0 and 1 instead of true/false.
```suggestion
            $('#dp2p_info').setVisibility(1);
```